apis/Google.Cloud.AutoML.V1/docs/history.md

```diff
@@ -0,0 +1,27 @@
+# Version history
+
+# Version 1.0.0, released 2019-12-10
+
+First GA release. No API surface changes since 1.0.0-beta03.
+
+- [Commit 201d00f](https://github.com/googleapis/google-cloud-dotnet/commit/201d00f): Descriptor-only changes
```

Comment:
Maybe consider removing these two, although I don't feel strongly about it

Reply:
Well spotted - had intended to :) Will do so now, then merge.